components/MenuSearch/MenuSearch.vue

```diff
@@ -89,10 +87,24 @@
 <script setup lang="ts">
 import { RiArrowRightSLine, RiDatabase2Line, RiGovernmentLine, RiLineChartLine, RiRobot2Line, RiSearchLine } from '@remixicon/vue'
 import { Combobox, ComboboxButton, ComboboxInput, ComboboxOption, ComboboxOptions, TransitionRoot } from '@headlessui/vue'
+import type { Component } from 'vue'
+
+type Item = {
+  icon: Component
+  type: string
+  to: string
+  external: boolean
+}
 
 const { t } = useI18n()
 const localePath = useLocalePath()
 const query = ref('')
+const selectedItem = ref<null | Item>(null)
+
+watch(selectedItem, () => {
```

Comment:
I think we also navigate if we click out of the menu?

Reply:
I don't reproduce the problem… Are you sure?

Reply:
Seems to be a bug with Headless UI https://github.com/tailwindlabs/headlessui/issues/3615 but trying to fix it…

Reply:
Loved the one hour investigation for the one line fix :-) 0e1ab48